piwheels/slave/builder.py

```diff
@@ -531,19 +563,14 @@ def build_dependencies(self, wheel):
             providers = {
                 pkg.name for pkg in apt_cache
                 if pkg.installed is not None
-                and lib in pkg.installed_files}
+                and lib in pkg.installed_files
+            }
             assert len(providers) <= 1
-            try:
-                deps['apt'].add(providers.pop())
-            except KeyError:
-                deps[''].add(lib)
+            apt_dependencies |= providers
```

Comment:
Not sure we want to remove the tracking of libraries with no associated package (even if this data doesn't wind up in the database)

Reply:
I've reverted this change